frontend/benefit/applicant/src/components/applications/forms/application/step5/useApplicationFormStep5.ts

```diff
@@ -0,0 +1,85 @@
+import hdsToast from 'benefit/applicant/components/toast/Toast';
+import ApplicationContext from 'benefit/applicant/context/ApplicationContext';
+import useUpdateApplicationQuery from 'benefit/applicant/hooks/useUpdateApplicationQuery';
+import { useTranslation } from 'benefit/applicant/i18n';
+import {
+  Application,
+  ApplicationData,
+} from 'benefit/applicant/types/application';
+import { getApplicationStepString } from 'benefit/applicant/utils/common';
+import { TFunction } from 'next-i18next';
+import React, { useEffect, useState } from 'react';
+import snakecaseKeys from 'snakecase-keys';
+
+type ExtendedComponentProps = {
+  t: TFunction;
+  handleNext: () => void;
+  handleBack: () => void;
+  translationsBase: string;
+};
+
+const useApplicationFormStep5 = (
+  application: Application
+): ExtendedComponentProps => {
+  const translationsBase = 'common:applications.sections.credentials.sections';
+  const { applicationTempData, setApplicationTempData } =
+    React.useContext(ApplicationContext);
+  const { t } = useTranslation();
+  const [step, setStep] = useState<number>(3);
```

Comment:
Do we really need to store the step in the state? Why don't we just pass the number as constant since the backend is the source of truth?